File: PartyApi/Models/Activity.cs
using System;
using System.Collections.Generic;

namespace PartyApi.Models
{
    public partial class Activity
    {
        public int Id { get; set; }
        public int PartyId { get; set; }
        public int UserId { get; set; }
        public DateTime ApplyDate { get; set; }
        public int ActAmt { get; set; }
        public int IsWaiting { get; set; }
        public string BankName { get; set; }
        public string BankNumber6 { get; set; }
        public string BankDate { get; set; }
        public string FriendsName { get; set; }
        public int MyNo { get; set; }
        public int CheckOver { get; set; }
        public string RetrunNote { get; set; }
        public string Notes { get; set; }
        public int? WriteType { get; set; }
        public DateTime? WriteTime { get; set; }
        public string WriteUser { get; set; }
        public string WriteIp { get; set; }

        public virtual Party Party { get; set; }
        public virtual Member User { get; set; }
    }
}

File: PartySpa/src/app/_shared/interface/activity.ts
// import { UseExistingWebDriver } from 'protractor/built/driverProviders';
// import { User } from './user';
// import { Party } from './party';

export interface Activity {
  id: number;
  partyId: number;
  userId: number;
  applyDate: Date;
  actAmt: number;
  bankName: string;
  bankNumber6: string;
  bankDate: string;
  friendsName: string;
  myNo: number;
  checkOver: number;
  notes: string;
  retrunNote: string;

  // user: User;
  // party: Party;
  firstName: string;
  lastName: string;
  sex: number;
  birthYear: number;
  mainPhotoUrl: string;
  education: number;
  marry: number;

  // 以下為活動檔的資料
  partyName: string;
  partyDate: Date;
}
